token/core/fabtoken/auditor.go

```diff
@@ -8,6 +8,7 @@ package fabtoken
 
 import "github.com/hyperledger-labs/fabric-token-sdk/token/driver"
 
+// AuditorCheck verifies if the passed tokenRequest matches the tokenRequestMetadata
 func (s *Service) AuditorCheck(tokenRequest *driver.TokenRequest, tokenRequestMetadata *driver.TokenRequestMetadata, txID string) error {
 	// TODO:
```

Comment:
address this todo (maybe open an issue?)

Reply:
done